numpy/ma/core.py

```diff
@@ -6824,7 +6824,16 @@ def argsort(a, axis=np._NoValue, kind=None, order=None, endwith=True, fill_value
 argsort.__doc__ = MaskedArray.argsort.__doc__
 
 def sort(a, axis=-1, kind=None, order=None, endwith=True, fill_value=None):
-    "Function version of the eponymous method."
+    """
+    Return a sorted copy of the masked array.
+
+    Equivalent to creating an copy of the array
+    and applying MaskedArray sort() method.
+
+    See Also
+    --------
+    numpy.ma.MaskedArray.sort()
```

Comment:
```suggestion
    MaskedArray.sort : equivalent function
```
to make it consistent with other docstrings.

Reply:
Addressed the changes.